Automattic/wp-calypso
# Mailing Lists: Add description for the 'promotion' category.

This change adds a formal description to the 'promotion' mailing list category for WordPress.com. It also updates the description for the Jetpack-specific promotion category so that the two share a similar description. 

Currently, if a user unsubscribes from this list, it doesn't provide as much information as it should (and does not match up with the other mailing list categories, which are properly configured): 

<img width="773" alt="Screen Shot 2020-08-07 at 12 18 12 PM" src="https://user-images.githubusercontent.com/625866/89666484-722eda80-d8a8-11ea-80a0-8666d3c84674.png">

#### Changes proposed in this Pull Request

* Add a formal title to the "promotion" category; this essentially allows the "P" to be properly capitalized.
* Add a formal description to the "promotion" category.
* Update the description of the "jetpack_promotion" category so that it mirrors the new value assigned to the "promotion" (WordPress.com) category.

#### Testing instructions

1. Visit the unsubscribe URLs provided below.
2. Confirm that the unsubscribe (and resubscribe) action works, as expected.
3. Confirm that the title and description of each message type is correctly updated, according to the code.

* http://calypso.localhost:3000/mailing-lists/unsubscribe?category=promotion&email=codefourcomics%40gmail.com&hmac=625efe95d31cc740b44272d3d605d567
* http://calypso.localhost:3000/mailing-lists/unsubscribe?category=jetpack_promotion&email=codefourcomics%40gmail.com&hmac=2601310d75f189c6f7d981b0c8a199cf

Here is what you should see:

<img width="854" alt="Screen Shot 2020-08-07 at 12 25 52 PM" src="https://user-images.githubusercontent.com/625866/89666913-2fb9cd80-d8a9-11ea-9fd5-88ff21d5605a.png">
<img width="803" alt="Screen Shot 2020-08-07 at 12 25 59 PM" src="https://user-images.githubusercontent.com/625866/89666911-2f213700-d8a9-11ea-9b86-b6b3593748d7.png">
